<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QChanDetails</class>
 <widget class="QWidget" name="QChanDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>57</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="lIcon">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="lChan">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lProgram">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Program</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lFromTo">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>from ... to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="timeProgess">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>5</height>
        </size>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
